docset/windows/storage/reset-physicaldisk.md

```diff
@@ -54,18 +54,20 @@ If you run **Reset-PhysicalDisk** on a disk that is not a part of a Storage pool
 ## EXAMPLES
 
 ### EXAMPLE 1
-```
+```powershell
 PS C:\> Reset-PhysicalDisk -FriendlyName "PhysicalDisk5"
 ```
 
 This example resets the state of a specific physical disk.
 
 ### EXAMPLE 2
+```powershell
+$BadDisks = (Get-Physicaldisk | Where-Object -FilterScript {$_.HealthStatus -Eq "Unhealthy"}
```

Comment:
There appears to be an unbalanced bracket here.

Reply:
I have updated that!